forch/faucetizer.py

```diff
@@ -101,7 +101,7 @@ def _faucetize(self):
 
                 port_cfg['native_vlan'] = vid
                 if device.behavior.role:
-                    port_cfg['acls_in'] = [f'role_{device.behavior.role}']
+                    port_cfg['acls_in'] = [f'role_{device.behavior.role}', 'allow_all']
```

Comment:
the tail shouldn't be "allow_all" but rather "tail_acl" or something like that -- since, well, what happens if somebody puts something in there that's not allow all?  Here in the code, it needs to describe what it actually is (the acl placed at the end), not so specific that it might be wrong if configuration changes.